api/organization.go

```diff
@@ -163,22 +164,39 @@ func (a *OrganizationAPI) SyncOrganizations(c *gin.Context) {
 	if token == "" {
 		c.JSON(http.StatusBadRequest, common.ErrorResponse{
 			Code:    http.StatusBadRequest,
-			Message: "user's github token is not set",
+			Message: "user's scm token is not set",
 		})
 
 		return
 	}
-
-	err = a.githubImporter.ImportOrganizationsFromGithub(user, token)
-	if err != nil {
-		errorHandler.Handle(err)
-
-		c.JSON(http.StatusInternalServerError, common.ErrorResponse{
-			Code:    http.StatusInternalServerError,
-			Message: "syncronization failed",
-			Error:   err.Error(),
-		})
-
+	switch provider {
```

Comment:
Error handling could be moved to a commonplace, since it's the same, to the bottom of the switch case for example.

Reply:
done